src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Query/PropertyPages/IPropertyPageQueryCache.cs

```diff
@@ -41,5 +41,6 @@ internal interface IPropertyPageQueryCache
     {
         Task<IRule?> BindToRule(ProjectConfiguration projectConfiguration, string schemaName);
         Task<IImmutableSet<ProjectConfiguration>?> GetKnownConfigurationsAsync();
+        Task<ProjectConfiguration?> GetDefaultConfigurationAsync();
```

Comment:
>GetDefaultConfigurationAsync [](start = 36, length = 28)

wouldn't GetSuggestedConfigurationAsync be a more accurate name for this method? Defaults sound like things that don't need to be computed

Reply:
Updated.